hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/subscription/id/NotificationAwareSubscriptionIdsCache.java

```diff
@@ -40,12 +73,14 @@ private void putSubscriptionId(SubscriptionName name) {
 
     @Override
     public Optional<SubscriptionId> getSubscriptionId(SubscriptionName subscriptionName) {
-        return Optional.ofNullable(nameToIdMap.get(subscriptionName));
+        return Optional.ofNullable(Optional.ofNullable(nameToIdMap.get(subscriptionName))
```

Comment:
Dunno, but maybe a static import for `ofNullable` isn't a bad idea at all, wdyt?